docs/deployments.md

```diff
@@ -17,3 +19,46 @@ If we use the Jira issue keys (`JIRA-*`) in the commit messages as shown in the
 To create deployments as a part of a GitHub Actions workflow, you can use [this deployment action](https://github.com/chrnorm/deployment-action), for example. Use the action `chrnorm/deployment-action@releases/v1` to create a deployment and don't forget to use the `chrnorm/deployment-status@releases/v1` to update the state of a deployment.
 
 **The GitHub for Jira app only listens to `deployment_status` events**. Just creating a deployment via the [create deployment API](https://docs.github.com/en/rest/reference/repos#create-a-deployment) (or the `chrnorm/deployment-action@releases/v1` action) isn't enough. After creating a deployment you need to call the [create deployment status API](https://docs.github.com/en/rest/reference/repos#create-a-deployment-status) (or the `chrnorm/deployment-status@releases/v1` action) at least once to provide a status for that deployment (success, failure, pending, ...).
+
+
+## Environment mapping
```

Comment:
Read this before reviewing the rest, as it hopefully clarifies the intent of this feature. 

Link for better reading: https://github.com/atlassian/github-for-jira/blob/user-config-2/docs/deployments.md#environment-mapping